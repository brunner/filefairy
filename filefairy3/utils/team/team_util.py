#!/usr/bin/env python
# -*- coding: utf-8 -*-

_collection = {
    '31': 'ARI',
    '32': 'ATL',
    '33': 'BAL',
    '34': 'BOS',
    '35': 'CWS',
    '36': 'CHC',
    '37': 'CIN',
    '38': 'CLE',
    '39': 'COL',
    '40': 'DET',
    '41': 'MIA',
    '42': 'HOU',
    '43': 'KC',
    '44': 'LAA',
    '45': 'LAD',
    '46': 'MIL',
    '47': 'MIN',
    '48': 'NYY',
    '49': 'NYM',
    '50': 'OAK',
    '51': 'PHI',
    '52': 'PIT',
    '53': 'SD',
    '54': 'SEA',
    '55': 'SF',
    '56': 'STL',
    '57': 'TB',
    '58': 'TEX',
    '59': 'TOR',
    '60': 'WAS',
}


def abbreviation(teamid):
    return _collection.get(teamid, '')


def divisions():
    return [
        ('AL East', ('33', '34', '48', '57', '59')),
        ('AL Central', ('35', '38', '40', '43', '47')),
        ('AL West', ('42', '44', '50', '54', '58')),
        ('NL East', ('32', '41', '49', '51', '60')),
        ('NL Central', ('36', '37', '46', '52', '56')),
        ('NL West', ('31', '39', '45', '53', '55')),
    ]
