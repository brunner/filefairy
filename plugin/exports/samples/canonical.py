#!/usr/bin/env python3
# -*- coding: utf-8 -*-

import os
import re
import sys

_path = os.path.dirname(os.path.abspath(__file__))
_root = re.sub(r'/plugin/exports/samples', '', _path)
sys.path.append(_root)
from common.elements.elements import card  # noqa
from common.elements.elements import cell  # noqa
from common.elements.elements import col  # noqa
from common.elements.elements import span  # noqa
from common.elements.elements import table  # noqa
from util.team.team import logo_absolute  # noqa

_t31 = [
    cell(content=logo_absolute('31', 'Arizona', 'left')),
    cell(content='5 - 5'),
    cell(content='W1')
]
_t32 = [
    cell(content=logo_absolute('32', 'Atlanta', 'left')),
    cell(content='10 - 0'),
    cell(content='W10')
]
_t33 = [
    cell(content=logo_absolute('33', 'Baltimore', 'left')),
    cell(content='6 - 4'),
    cell(content='W1')
]
_t34 = [
    cell(content=logo_absolute('34', 'Boston', 'left')),
    cell(content='10 - 0'),
    cell(content='W10')
]
_t35 = [
    cell(content=logo_absolute('35', 'Chicago', 'left')),
    cell(content='9 - 1'),
    cell(content='L1')
]
_t36 = [
    cell(content=logo_absolute('36', 'Chicago', 'left')),
    cell(content='8 - 2'),
    cell(content='L1')
]
_t37 = [
    cell(content=logo_absolute('37', 'Cincinnati', 'left')),
    cell(content='10 - 0'),
    cell(content='W10')
]
_t38 = [
    cell(content=logo_absolute('38', 'Cleveland', 'left')),
    cell(content='8 - 2'),
    cell(content='W2')
]
_t39 = [
    cell(content=logo_absolute('39', 'Colorado', 'left')),
    cell(content='10 - 0'),
    cell(content='W10')
]
_t40 = [
    cell(content=logo_absolute('40', 'Detroit', 'left')),
    cell(content='10 - 0'),
    cell(content='W10')
]
_t41 = [
    cell(content=logo_absolute('41', 'Miami', 'left')),
    cell(content='4 - 6'),
    cell(content='W1')
]
_t42 = [
    cell(content=logo_absolute('42', 'Houston', 'left')),
    cell(content='10 - 0'),
    cell(content='W10')
]
_t43 = [
    cell(content=logo_absolute('43', 'Kansas City', 'left')),
    cell(content='9 - 1'),
    cell(content='W5')
]
_t44 = [
    cell(content=logo_absolute('44', 'Los Angeles', 'left')),
    cell(content='7 - 3'),
    cell(content='L1')
]
_t45 = [
    cell(content=logo_absolute('45', 'Los Angeles', 'left')),
    cell(content='5 - 5'),
    cell(content='W2')
]
_t46 = [
    cell(content=logo_absolute('46', 'Milwaukee', 'left')),
    cell(content='10 - 0'),
    cell(content='W10')
]
_t47 = [
    cell(content=logo_absolute('47', 'Minnesota', 'left')),
    cell(content='10 - 0'),
    cell(content='W10')
]
_t48 = [
    cell(content=logo_absolute('48', 'New York', 'left')),
    cell(content='10 - 0'),
    cell(content='W10')
]
_t49 = [
    cell(content=logo_absolute('49', 'New York', 'left')),
    cell(content='8 - 2'),
    cell(content='L1')
]
_t50 = [
    cell(content=logo_absolute('50', 'Oakland', 'left')),
    cell(content='7 - 3'),
    cell(content='L3')
]
_t51 = [
    cell(content=logo_absolute('51', 'Philadelphia', 'left')),
    cell(content='3 - 7'),
    cell(content='L5')
]
_t52 = [
    cell(content=logo_absolute('52', 'Pittsburgh', 'left')),
    cell(content='6 - 4'),
    cell(content='L1')
]
_t53 = [
    cell(content=logo_absolute('53', 'San Diego', 'left')),
    cell(content='9 - 1'),
    cell(content='W8')
]
_t54 = [
    cell(content=logo_absolute('54', 'Seattle', 'left')),
    cell(content='10 - 0'),
    cell(content='W10')
]
_t55 = [
    cell(content=logo_absolute('55', 'San Francisco', 'left')),
    cell(content='2 - 8'),
    cell(content='L4')
]
_t56 = [
    cell(content=logo_absolute('56', 'St. Louis', 'left')),
    cell(content='10 - 0'),
    cell(content='W10')
]
_t57 = [
    cell(content=logo_absolute('57', 'Tampa Bay', 'left')),
    cell(content='10 - 0'),
    cell(content='W10')
]
_t58 = [
    cell(content=logo_absolute('58', 'Texas', 'left')),
    cell(content='9 - 1'),
    cell(content='W9')
]
_t59 = [
    cell(content=logo_absolute('59', 'Toronto', 'left')),
    cell(content='9 - 1'),
    cell(content='W4')
]
_t60 = [
    cell(content=logo_absolute('60', 'Washington', 'left')),
    cell(content='5 - 5'),
    cell(content='L1')
]

_breadcrumbs = [{
    'href': '/',
    'name': 'Fairylab'
}, {
    'href': '',
    'name': 'Exports'
}]

_tcols = [col()] + [col(clazz='text-center')] * 5
_table = table(
    clazz='table-sm',
    hcols=_tcols,
    bcols=_tcols,
    body=[[
        cell(content='AL East'),
        cell(content='BAL'),
        cell(content='BOS'),
        cell(content='NYY'),
        cell(content='TB'),
        cell(content='TOR')
    ], [
        cell(content='AL Central'),
        cell(content=span(['text-success', 'border', 'px-1'], 'CWS')),
        cell(content='CLE'),
        cell(content=span(['text-success', 'border', 'px-1'], 'DET')),
        cell(content='KC'),
        cell(content=span(['text-success', 'border', 'px-1'], 'MIN'))
    ], [
        cell(content='AL West'),
        cell(content='HOU'),
        cell(content=span(['text-success', 'border', 'px-1'], 'LAA')),
        cell(content='OAK'),
        cell(content=span(['text-success', 'border', 'px-1'], 'SEA')),
        cell(content=span(['text-success', 'border', 'px-1'], 'TEX'))
    ], [
        cell(content='NL East'),
        cell(content=span(['text-success', 'border', 'px-1'], 'ATL')),
        cell(content='MIA'),
        cell(content=span(['text-success', 'border', 'px-1'], 'NYM')),
        cell(content='PHI'),
        cell(content=span(['text-success', 'border', 'px-1'], 'WAS'))
    ], [
        cell(content='NL Cental'),
        cell(content=span(['text-success', 'border', 'px-1'], 'CHC')),
        cell(content=span(['text-success', 'border', 'px-1'], 'CIN')),
        cell(content=span(['text-success', 'border', 'px-1'], 'MIL')),
        cell(content='PIT'),
        cell(content=span(['text-success', 'border', 'px-1'], 'STL'))
    ], [
        cell(content='NL West'),
        cell(content='ARI'),
        cell(content='COL'),
        cell(content=span(['text-success', 'border', 'px-1'], 'LAD')),
        cell(content=span(['text-success', 'border', 'px-1'], 'SD')),
        cell(content='SF')
    ]])

_breakdown = [
    span(['text-success', 'border', 'px-1'], '16 new'), '14 old',
    span(['text-secondary'], '0 ai')
]
_info = 'Upcoming sim contains ' + ', '.join(_breakdown) + '.'
_live = card(
    title='53%', info=_info, table=_table, ts='06:02:30 EDT (1985-10-26)')

_scols = [
    col(clazz='position-relative'),
    col(clazz='text-center w-25'),
    col(clazz='text-center w-25')
]
_standings = [
    table(
        clazz='border mt-3',
        hcols=_scols,
        bcols=_scols,
        head=[
            cell(content='AL East'),
            cell(content='Last 10'),
            cell(content='Streak')
        ],
        body=[_t34, _t48, _t57, _t59, _t33]),
    table(
        clazz='border mt-3',
        hcols=_scols,
        bcols=_scols,
        head=[
            cell(content='AL Central'),
            cell(content='Last 10'),
            cell(content='Streak')
        ],
        body=[_t40, _t47, _t43, _t35, _t38]),
    table(
        clazz='border mt-3',
        hcols=_scols,
        bcols=_scols,
        head=[
            cell(content='AL West'),
            cell(content='Last 10'),
            cell(content='Streak')
        ],
        body=[_t42, _t54, _t58, _t44, _t50]),
    table(
        clazz='border mt-3',
        hcols=_scols,
        bcols=_scols,
        head=[
            cell(content='NL East'),
            cell(content='Last 10'),
            cell(content='Streak')
        ],
        body=[_t32, _t49, _t60, _t41, _t51]),
    table(
        clazz='border mt-3',
        hcols=_scols,
        bcols=_scols,
        head=[
            cell(content='NL Central'),
            cell(content='Last 10'),
            cell(content='Streak')
        ],
        body=[_t37, _t46, _t56, _t36, _t52]),
    table(
        clazz='border mt-3',
        hcols=_scols,
        bcols=_scols,
        head=[
            cell(content='NL West'),
            cell(content='Last 10'),
            cell(content='Streak')
        ],
        body=[_t39, _t53, _t45, _t31, _t55])
]

subtitle = ''

tmpl = 'exports.html'

context = {
    'title': 'exports',
    'breadcrumbs': _breadcrumbs,
    'live': _live,
    'standings': _standings
}
