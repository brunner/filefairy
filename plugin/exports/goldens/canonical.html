<!doctype html>
<html lang="en">

<head>

<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link rel="stylesheet"
      href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
      integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
      crossorigin="anonymous">
<link rel="stylesheet"
      href="https://cdnjs.cloudflare.com/ajax/libs/open-iconic/1.1.1/font/css/open-iconic-bootstrap.min.css">
<title>Fairylab | exports</title>

<style>
html {
  position: relative;
  min-height: 100%;
}
body {
  margin-bottom: 60px;
}
header {
  background-color: #f5f5f5;
  width: 100%;
}
a.disabled{
  pointer-events: none;
  cursor: default;
}
.alert, .dropdown-toggle, .form-control {
  padding: .75rem;
}
.alert .oi {
  line-height: 24px;
  padding-right: .75rem;
}
.alert-danger {
  border: 1px solid #f5c6cb;
}
.alert-primary {
  border: 1px solid #b8daff;
}
.alert-success {
  border: 1px solid #c3e6cb;
}
.anchor {
  padding-top: 0px;
  margin-top: 0px;
  transition: padding-top .35s ease, margin-top .35s ease;
}
.badge {
  border-radius: 24px;
  height: 24px;
  line-height: 16px;
  text-align: center;
  width: 24px;
}
.breadcrumb {
  background-color: inherit;
  line-height: 60px;
  max-width: 468px;
  overflow-x: auto;
}
.dropdown-toggle {
  color: #6c767c;
}
.flex-grow {
  flex-grow: 1;
}
.left-8p {
  left: 8px;
}
.mr-12p {
  margin-right: 12px;
}
.nav-pills {
  overflow-x: auto;
  overflow-y: hidden;
}
.navbar {
  padding: 0;
}
.nav-pills .nav-item {
  margin: 0 .25rem;
  min-width: 32px;
  text-align: center;
}
.nav-pills .nav-link {
  padding: .25rem 0;
}
.nav-pills .nav-link:not(.active) {
  border: 1px solid #dee2e6;
}
.pl-24p {
  padding-left: 24px;
}
.pr-24p {
  padding-right: 24px;
}
.right-8p {
  right: 8px;
}
.table, .table-sm {
  margin-bottom: 0;
  word-break: break-word;
}
.table .thead-light td {
  background-color: #f5f5f5;
}
.table-fixed {
  table-layout: fixed;
}
.table-sm td {
  border-top: 0;
  padding: 0 .3rem 0 0;
}
.table .td-sm {
  padding: .75rem .25rem;
}
.table.collapse.show {
  display: table;
}
.top-14p {
  top: 14px;
}
.w-10 {
  width: 10%;
}
.w-20 {
  width: 20%;
}
.w-40 {
  width: 40%;
}
.w-55p {
  width: 55px;
}
.w-65p {
  width: 65px;
}
.w-75p {
  width: 75px;
}
.w-105p {
  width: 105px;
}
</style>
</head>

<body>

<!-- ^(base.header) -->
<header class="border-bottom sticky-top">
<div class="container">
<nav class="navbar navbar-light border-bottom" aria-label="nav">
<ol class="breadcrumb flex-nowrap text-truncate m-0 p-0">
<li class="breadcrumb-item">
<a href="/">Fairylab</a>
</li>
<li class="breadcrumb-item active" aria-current="page">Exports</li>
</ol>
</nav>

<div class="my-1">
<small class="text-muted">Last reloaded: 06:02:30 PDT (1985-10-26)</small>
</div>

</div>
</header>
<!-- $(base.header) -->

<!-- ^(base.main) -->
<main role="main" class="pb-3">

<!-- ^(exports.tracker) -->
<div class="container anchor" id="tracker">

<!-- ^(common.heading) -->
<div class="row">
<div class="col-12">
<div class="alert alert-secondary mt-3 mb-0">
<span class="oi oi-cloudy text-secondary"></span>Tracker
</div>
</div>
</div>
<!-- $(common.heading) -->

<!-- ^(exports.row) -->
<div class="row">

<!-- ^(exports.column) -->
<div class="col-md-6">

<!-- ^(common.table) -->
<table class="table border mt-3">

<!-- ^(common.table.head) -->
<thead class="thead-light">
<tr>
<td scope="col"class="position-relative">AL East</td>
<td scope="col" class="text-center w-25">Last 10</td>
<td scope="col" class="text-center w-25">Streak</td>
</tr>
</thead>
<!-- $(common.table.head) -->

<!-- ^(common.table.body) -->
<tbody>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/boston_red_sox_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Boston</span></td>
<td class="text-center w-25">10 - 0</td>
<td class="text-center w-25">W10</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/new_york_yankees_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">New York</span></td>
<td class="text-center w-25">10 - 0</td>
<td class="text-center w-25">W10</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/tampa_bay_rays_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Tampa Bay</span></td>
<td class="text-center w-25">10 - 0</td>
<td class="text-center w-25">W10</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/toronto_blue_jays_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Toronto</span></td>
<td class="text-center w-25">9 - 1</td>
<td class="text-center w-25">W4</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/baltimore_orioles_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Baltimore</span></td>
<td class="text-center w-25">6 - 4</td>
<td class="text-center w-25">W1</td>
</tr>
</tbody>
<!-- $(common.table.body) -->

</table>
<!-- $(common.table) -->

<!-- ^(common.table) -->
<table class="table border mt-3">

<!-- ^(common.table.head) -->
<thead class="thead-light">
<tr>
<td scope="col"class="position-relative">AL Central</td>
<td scope="col" class="text-center w-25">Last 10</td>
<td scope="col" class="text-center w-25">Streak</td>
</tr>
</thead>
<!-- $(common.table.head) -->

<!-- ^(common.table.body) -->
<tbody>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/detroit_tigers_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Detroit</span></td>
<td class="text-center w-25">10 - 0</td>
<td class="text-center w-25">W10</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/minnesota_twins_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Minnesota</span></td>
<td class="text-center w-25">10 - 0</td>
<td class="text-center w-25">W10</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/kansas_city_royals_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Kansas City</span></td>
<td class="text-center w-25">9 - 1</td>
<td class="text-center w-25">W5</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/chicago_white_sox_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Chicago</span></td>
<td class="text-center w-25">9 - 1</td>
<td class="text-center w-25">L1</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/cleveland_indians_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Cleveland</span></td>
<td class="text-center w-25">8 - 2</td>
<td class="text-center w-25">W2</td>
</tr>
</tbody>
<!-- $(common.table.body) -->

</table>
<!-- $(common.table) -->

<!-- ^(common.table) -->
<table class="table border mt-3">

<!-- ^(common.table.head) -->
<thead class="thead-light">
<tr>
<td scope="col"class="position-relative">AL West</td>
<td scope="col" class="text-center w-25">Last 10</td>
<td scope="col" class="text-center w-25">Streak</td>
</tr>
</thead>
<!-- $(common.table.head) -->

<!-- ^(common.table.body) -->
<tbody>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/houston_astros_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Houston</span></td>
<td class="text-center w-25">10 - 0</td>
<td class="text-center w-25">W10</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/seattle_mariners_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Seattle</span></td>
<td class="text-center w-25">10 - 0</td>
<td class="text-center w-25">W10</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/texas_rangers_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Texas</span></td>
<td class="text-center w-25">9 - 1</td>
<td class="text-center w-25">W9</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/los_angeles_angels_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Los Angeles</span></td>
<td class="text-center w-25">7 - 3</td>
<td class="text-center w-25">L1</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/oakland_athletics_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Oakland</span></td>
<td class="text-center w-25">7 - 3</td>
<td class="text-center w-25">L3</td>
</tr>
</tbody>
<!-- $(common.table.body) -->

</table>
<!-- $(common.table) -->

</div>
<!-- $(exports.column) -->

<!-- ^(exports.column) -->
<div class="col-md-6">

<!-- ^(common.table) -->
<table class="table border mt-3">

<!-- ^(common.table.head) -->
<thead class="thead-light">
<tr>
<td scope="col"class="position-relative">NL East</td>
<td scope="col" class="text-center w-25">Last 10</td>
<td scope="col" class="text-center w-25">Streak</td>
</tr>
</thead>
<!-- $(common.table.head) -->

<!-- ^(common.table.body) -->
<tbody>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/atlanta_braves_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Atlanta</span></td>
<td class="text-center w-25">10 - 0</td>
<td class="text-center w-25">W10</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/new_york_mets_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">New York</span></td>
<td class="text-center w-25">8 - 2</td>
<td class="text-center w-25">L1</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/washington_nationals_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Washington</span></td>
<td class="text-center w-25">5 - 5</td>
<td class="text-center w-25">L1</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/miami_marlins_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Miami</span></td>
<td class="text-center w-25">4 - 6</td>
<td class="text-center w-25">W1</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/philadelphia_phillies_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Philadelphia</span></td>
<td class="text-center w-25">3 - 7</td>
<td class="text-center w-25">L5</td>
</tr>
</tbody>
<!-- $(common.table.body) -->

</table>
<!-- $(common.table) -->

<!-- ^(common.table) -->
<table class="table border mt-3">

<!-- ^(common.table.head) -->
<thead class="thead-light">
<tr>
<td scope="col"class="position-relative">NL Central</td>
<td scope="col" class="text-center w-25">Last 10</td>
<td scope="col" class="text-center w-25">Streak</td>
</tr>
</thead>
<!-- $(common.table.head) -->

<!-- ^(common.table.body) -->
<tbody>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/cincinnati_reds_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Cincinnati</span></td>
<td class="text-center w-25">10 - 0</td>
<td class="text-center w-25">W10</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/milwaukee_brewers_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Milwaukee</span></td>
<td class="text-center w-25">10 - 0</td>
<td class="text-center w-25">W10</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/st_louis_cardinals_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">St. Louis</span></td>
<td class="text-center w-25">10 - 0</td>
<td class="text-center w-25">W10</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/chicago_cubs_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Chicago</span></td>
<td class="text-center w-25">8 - 2</td>
<td class="text-center w-25">L1</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/pittsburgh_pirates_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Pittsburgh</span></td>
<td class="text-center w-25">6 - 4</td>
<td class="text-center w-25">L1</td>
</tr>
</tbody>
<!-- $(common.table.body) -->

</table>
<!-- $(common.table) -->

<!-- ^(common.table) -->
<table class="table border mt-3">

<!-- ^(common.table.head) -->
<thead class="thead-light">
<tr>
<td scope="col"class="position-relative">NL West</td>
<td scope="col" class="text-center w-25">Last 10</td>
<td scope="col" class="text-center w-25">Streak</td>
</tr>
</thead>
<!-- $(common.table.head) -->

<!-- ^(common.table.body) -->
<tbody>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/colorado_rockies_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Colorado</span></td>
<td class="text-center w-25">10 - 0</td>
<td class="text-center w-25">W10</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/san_diego_padres_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">San Diego</span></td>
<td class="text-center w-25">9 - 1</td>
<td class="text-center w-25">W8</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/los_angeles_dodgers_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Los Angeles</span></td>
<td class="text-center w-25">5 - 5</td>
<td class="text-center w-25">W2</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/arizona_diamondbacks_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">Arizona</span></td>
<td class="text-center w-25">5 - 5</td>
<td class="text-center w-25">W1</td>
</tr>
<tr>
<td scope="row"class="position-relative"><img src="https://orangeandblueleaguebaseball.com/StatsLab/reports/news/html/images/team_logos/san_francisco_giants_40.png" width="20" height="20" border="0" class="position-absolute left-8p top-14p"><span class="align-middle d-block text-truncate pl-24p">San Francisco</span></td>
<td class="text-center w-25">2 - 8</td>
<td class="text-center w-25">L4</td>
</tr>
</tbody>
<!-- $(common.table.body) -->

</table>
<!-- $(common.table) -->

</div>
<!-- $(exports.column) -->

</div>
<!-- $(exports.row) -->

</div>
<!-- $(exports.tracker) -->

</main>
<!-- $(base.main) -->

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"
        integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/js/bootstrap.min.js"
        integrity="sha384-a5N7Y/aK3qNeh15eJKGWxsqtnX/wWdSZSKp+81YjTmS15nvnvxKHuzaWwXHDli+4"
        crossorigin="anonymous"></script>
<script>
$('.repl-toggler').on('click', function(e) {
  if ($(this).hasClass('disabled')) {
    e.preventDefault();
  } else {
    $(this).toggleClass('disabled text-secondary');
    $(this).html($(this).data('repl'));
  }
});
$('.show-toggler').on('click', function(e) {
  $(this).toggleClass('border-bottom-0');
});
</script>


</body>

</html>